Adding custom inventory script example (#1941)
ansible/aap-docs: downstream/modules/platform/ref-controller-scm-inv-source-fields.adoc

== Supported File Syntax

{ControllerNameStart} uses the `ansible-inventory` module from Ansible to process inventory files, and supports all valid inventory syntax that {ControllerName} requires.

> The following is an example of a working example of JSON output that {ControllerName} can read in:

Reviewer: ```suggestion
The following is a working example of JSON output that {ControllerName} can read for the import:
```

----
{
    "_meta": {
        "hostvars": {
            "host1": {
                "fly_rod": true
            }
        }
    },
    "all": {
        "children": [
            "groupA",
            "ungrouped"
        ]
    },
    "groupA": {
        "hosts": [
            "host1",
            "host10",
            "host11",
            "host12",
            "host13",
            "host14",
            "host15",
            "host16",
            "host17",
            "host18",
            "host19",
            "host2",
            "host20",
            "host21",
            "host22",
            "host23",
            "host24",
            "host25",
            "host3",
            "host4",
            "host5",
            "host6",
            "host7",
            "host8",
            "host9"
        ]
    }
} 

> ----

Reviewer: ```suggestion
----
For information on how to implement the inventory script, see support article: https://access.redhat.com/solutions/6997130.
```
